table/get_context.cc

```diff
@@ -102,7 +105,9 @@ void GetContext::MarkKeyMayExist() {
 
 void GetContext::SaveValue(const Slice& value, SequenceNumber /*seq*/) {
   assert(state_ == kNotFound);
-  appendToReplayLog(replay_log_, kTypeValue, value);
+  const std::string kMaxTs(TimestampSize(), '\xff');
+
```

Comment:
This version of the `SaveValue` is only called from the cuckoo_table_reader which does not have user defined timestamp support. So instead of saving the maximum timestmap, which takes some space, what about asserting timestamp size is always zero in this function and pass an empty `Slice` to `appendToReplayLog` as timestamp.